app/actions.tsx

```
@@ -359,147 +365,274 @@ async function submit(formData?: FormData, skip?: boolean) {
-                name: output.toolName,
-                type: 'tool'
-              }
-            ]
-          })
+    )
+    answer = fullResponse
+    toolOutputs = toolResponses
+    errorOccurred = hasError
+
+    if (toolOutputs.length > 0) {
+      toolOutputs.map(output => {
```

Biome (2.3.13)

[error] This callback passed to map() iterable method should always return a value.

Add a return with a value to this callback.

(lint/suspicious/useIterableCallbackReturn)